lib/commands/find.js

```diff
@@ -58,9 +60,16 @@ helpers.findElOrEls = async function (strategy, selector, mult, context = '') {
     }
   }
 
-  return element;
+  if (mult) {
```

Comment:
It does. I basically copied this from there. It should definitely be abstracted out (perhaps having the actual drivers just implement the actual find code, and the base driver doing the call implicit wait magics).